mistral/mixtral-8x7b-instruct-vllm/config.yaml

```diff
@@ -1,10 +1,8 @@
-base_image:
-  image: nvcr.io/nvidia/pytorch:23.11-py3
-  python_executable_path: /usr/bin/python3
+model_metadata: {}
 model_name: Mixtral 8x7B
 python_version: py310
 requirements:
-- vllm==0.2.4
+- vllm==0.2.5
```

Comment:
can you increase the `runtime: predict_concurrency: 128` or higher so that people get good concurrent perf with continuous batching out of the box?